package/yast2-network.spec

```diff
@@ -17,7 +17,7 @@
 
 
 Name:           yast2-network
-Version:        4.2.24
```

Comment:
Apparently I didn't increase the version in #997 when rebasing. So, it was supposed to be yast2-network-4.2.25 already and we bump it now to 4.2.26